Packages/com.unity.input/Runtime/Input.cs

```diff
@@ -66,6 +67,10 @@ public static partial class Input // Partial to allow us to inject project-speci
 
         internal static void OnEnteringPlayMode()
         {
+            // EnhancedTouch is required for the Touch API
+            if (!EnhancedTouchSupport.enabled)
+                EnhancedTouchSupport.Enable();
```

Comment:
ATM this thing functions like a counter. So just calling Enable here and Disable in Reset prob makes sense.